R/util.R

```diff
@@ -1437,6 +1437,11 @@ extract_from_date <- function(x, type = "fltoyear") {
     wdaylong = {
       ret <- lubridate::wday(x, label=TRUE, abbr=FALSE)
     },
+    daytype = {
```

Comment:
The name "daytype" sounds too generic for this concept. How about "wdaytype"?

Reply:
updated as wdaytype